addons/mail/static/src/discuss/core/public_web/discuss_command_palette.js

```diff
@@ -150,15 +150,19 @@ export class DiscussCommandPalette {
     buildResults(filtered) {
         const TOTAL_LIMIT = this.ui.isSmall ? 7 : 8;
         const remaining = TOTAL_LIMIT - (filtered ? filtered.size : 0);
-        let personas = Object.values(this.store.Persona.records).filter(
-            (persona) =>
-                persona.isInternalUser && cleanTerm(persona.name).includes(this.cleanedTerm)
-        );
-        personas = this.suggestion
-            .sortPartnerSuggestions(personas, this.cleanedTerm)
-            .slice(0, TOTAL_LIMIT)
-            .filter((persona) => !filtered || !filtered.has(persona));
```

Comment:
And even this line, why filter here after sort and slice, rather than using the existing filter even higher in the code?